solidity/src/CoinToss.sol

```diff
@@ -54,9 +54,8 @@ contract CoinToss is CadenceRandomConsumer {
         // delete the open request from the coinTosses mapping
         delete coinTosses[msg.sender];
 
-        // fulfill the random request
-        uint64 randomResult = _fulfillRandomness(uint32(requestId));
-        uint8 coinFace = uint8(randomResult % 2);
+        // fulfill the random request within the inclusive range [0, 1]
+        uint8 coinFace = uint8(_fulfillRandomInRange(requestId, uint64(0), uint64(1)));
```

Comment:
The code to the left is totally valid (in the case where the modulo is `2`, `4`, `8`..). Maybe the contract could use a different example if we want to showcase `_fulfillRandomInRange`, maybe rolling a dice so that the range is `[0..5]` ?

Reply:
My thought is we can include it here for demonstration at least until we get another example. I like the idea of an odd sided dice roll or maybe a roulette wheel.